tests/conftest.py

```diff
@@ -87,6 +92,46 @@ def values_str() -> TListValuesInputStr:
     return [uuid_str(), uuid_str(), uuid_str()]
 
 
+@pytest.fixture
+def dictionary_name() -> TDictionaryName:
+    return uuid_str()
+
+
+@pytest.fixture
+def dictionary_field() -> TDictionaryField:
```

Comment:
```suggestion
def dictionary_field_str() -> TDictionaryFieldStr:
```

Reply:
I can see renaming the fixture name, but the type `TDictionaryField` is `str|bytes`, which is correct (but not the narrowest) for this? I used the fixture in tests where the field was used, so I opted for the broader field type.

Reply:
If this is specifically returning a string value, it should be named and typed like that. The correct type will make testing easier, the reader knows what it does, and it makes it useful for testing specifically that we do the right thing with strings vs bytes.

You can also have `dictionary_field` which returns TDictionaryField and it's useful for that to be `return random.choice((uuid_bytes(), uuid_str()))`. That's useful for writing one test for things like set and get. It will randomly choose combinations of bytes and str and we don't need to write individual tests for it, with enough test runs we get full coverage.

Reply:
Fixed in 0e76b79. I changed the name as suggested but changed the type hint to just `str` since it is unambiguous here and introducing a new type hint seems like overkill